server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -2218,8 +2218,13 @@ public void testSnapshotRelocatingPrimary() throws Exception {
                         .put("compress", randomBoolean())
                         .put("chunk_size", randomIntBetween(100, 1000), ByteSizeUnit.BYTES)));
 
-        // Create index on 1 nodes and make sure each node has a primary by setting no replicas
-        assertAcked(prepareCreate("test-idx", 1, Settings.builder().put("number_of_replicas", 0)));
+        internalCluster().ensureAtLeastNumDataNodes(2);
```

Comment:
`prepareCreate` already does this I think

Reply:
Right it does -> removed, thanks :)